Update notification docs (#4373)
pipe-cd/pipecd: docs/content/en/docs/user-guide/managing-piped/configuration-reference.md

| Field | Type | Description | Required |
|-|-|-|-|

> | hookURL | string | The hookURL of a slack channel. | No |

Reviewer: We don't need to update this, `/docs` is for the current released version of pipecd (which need the hookURL to perform send message), when we release the new version which contains your patch, the `/docs` will be synced with `/docs-dev` and show the latest docs which fit the piped functional 😄

| Field | Type | Description | Required |
|-|-|-|-|
| hookURL | string | The hookURL of a slack channel. | No |

> | oauthToken | string | The token of a slack api. | No |

Reviewer: ```suggestion
| oauthToken | string | The token for slack api use. | No |
```
Would be best if you can also add a link to how to get slack oauth token docs.

Author: I updated it.
